src/node_api.cc

```diff
@@ -1002,6 +1002,27 @@ napi_status napi_get_property(napi_env env,
   return GET_RETURN_STATUS(env);
 }
 
+napi_status napi_delete_property(napi_env env,
+                                 napi_value object,
+                                 napi_value key,
+                                 bool* result) {
+  NAPI_PREAMBLE(env);
+  CHECK_ENV(env);
```

Comment:
`NAPI_PREAMBLE(env)` already calls `CHECK_ENV(env)`, so the latter is redundant here.